.github/workflows/ci.yml

```diff
@@ -2,16 +2,32 @@ name: CI
 
 on:
   push:
-    branches: [main]
+    branches: [ main ]
   pull_request:
-    branches: [main]
+    branches: [ main ]
 
 jobs:
-  test:
+  maven:
```

Comment:
Could you rename maven to junit? Thanks.

Reply:
done